### DISCUSSION TITLE
[Projects Beta] Embedded view

### DISCUSSION THREAD

**Discussion by rafaelugolini** [community/community#8355] [Projects and Issues]
It would be fantastic to have an embedded view to add projects over a notion page or similar products.

One product that is doing a great job at that is Airtble, check the [Embedding a view or base](https://support.airtable.com/hc/en-us/articles/217846478-Embedding-a-view-or-base) page documentation.

**Comment by wotta** [comment]
This would be a real nice addition indeed. I can imagine that there is some sort of difficulty that makes this harder.

Hope more information about this will get shared.
